Week-2/Assignment-1/index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html />
+<html>
+
+<head>
+    <meta charset="utf=8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Week-2 Assignment-1</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+
+    <header class="header">
+        <h1>What is Lorem Ipsum?</h1>
+        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the
+            industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and
+            scrambled it to make a type specimen book.</p>
+    </header>
+
+    <nav>
+        <ul class="nav">
+            <li><a href="https://www.lipsum.com/" target="_blank">Home</a></li>
+            <li><a href="https://www.lipsum.com/" target="_blank">Contact Us</a></li>
+            <li><a href="https://www.lipsum.com/" target="_blank">Careers</a></li>
+            <li><a href="https://www.lipsum.com/" target="_blank">About Us</a></li>
+        </ul>
+    </nav>
+
+    <main>
+        <figure>
+            <img src="https://www.w3schools.com/html/pic_trulli.jpg" alt="Trulli">
+            <figcaption>"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+                labore et dolore magna aliqua."</figcaption>
+        </figure>
+
+
+        <article>
```

Comment:
What is the difference between an article and a section ?

Reply:
The section tag is used to split the page into different sections like Introduction, Information, etc. Whereas the article tag contains independent content that doesn’t require any other context.